config/locales/flood_risk_engine/enrollment_mailer.en.yml

```diff
@@ -3,9 +3,9 @@ en:
   flood_risk_engine:
     enrollment_mailer:
       submitted:
-        subject: Registration submitted
+        subject: Flood risk activity exemption submitted
+        heading_html: 'Registration submitted: <br /> please wait for us to confirm it'
```

Comment:
I don't think you should be passing markup within a translation. I'd suggest breaking this down into elements and adding the break in the view code.